src/webrtc/groupCall.ts

```diff
@@ -609,6 +617,11 @@ export class GroupCall extends TypedEventEmitter<
      * @returns Whether muting/unmuting was successful
      */
     public async setLocalVideoMuted(muted: boolean): Promise<boolean> {
+        // Because we need a Local Call Feed to establish a call connection, we avoid muting video in case of empty
+        // video track. In this way we go sure if a client implements muting we don't raise an error.
+        if (this.initCallWithoutVideoAndAudio && this.localCallFeed?.stream.getVideoTracks().length === 0) {
+            return false;
+        }
```

Comment:
Sorry, it's somehow not clear to me what this piece of code does exactly

Reply:
My implementation allows feeds that does not contain tracks at all. I have to make sure that the user of the SDK doesn't try to mute a feed that doesn't have a video track. This case must be caught. It would be logically correct if we would have no LocalFeed. This case is covert below. But you need a LocalFeed to innit a GroupCall. I can't go the way of PTT either, because I have to assume that no device exists.

Reply:
Do we need to check for `this.initCallWithoutVideoAndAudio`? Wouldn't it always be an error to try & mute/unmute video if there was no video track?

Reply:
i'll remove it